include/serialport.h

```diff
@@ -51,28 +51,25 @@
 
 class MyFifo {
 public:
-	MyFifo(Bitu maxsize_) {
-		maxsize=size=maxsize_;
-		pos=used=0;
-		data=new Bit8u[size];
+	MyFifo(uint32_t maxsize_)
+	{
+		maxsize = size = maxsize_;
+		pos = used = 0;
+		data = new uint8_t[size];
 	}
 	~MyFifo() {
 		delete[] data;
 	}
-	INLINE Bitu getFree(void) {
-		return size-used;
-	}
+	INLINE uint32_t getFree(void) { return size - used; }
```

Comment:
Useless `void` parameter and redundant INLINE - I think we can get rid of both, and make this a cons method.

Reply:
I will take a pass through and try to catch both of these.  I wonder if the reformatter can snip out (void) function parameters?

Reply:
Cleaned them all up in commit 44f23961.